internal/service/pushbasedlogexport/data_source_schema.go

```diff
@@ -0,0 +1,61 @@
+package pushbasedlogexport
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func DataSourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"bucket_name": schema.StringAttribute{
+				Computed:            true,
+				Description:         "The name of the bucket to which the agent sends the logs to.",
+				MarkdownDescription: "The name of the bucket to which the agent sends the logs to.",
+			},
+			"create_date": schema.StringAttribute{
+				Computed:            true,
+				Description:         "Date and time that this feature was enabled on.",
```

Comment:
not for this PR, as we have the function to return the schema, we could have a helper function that for instance fill MarkdownDescription taking from Description if not filled so we only have one description in the code but it's filled for both Description and MarkdownDescription

Reply:
makes sense, adding @AgustinBettati for visibility. Right now I do see we have the same description in 4 places